src/main/resources/avro/ExternalMetric.avsc

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
I would recommend you switch over to https://github.com/Rackspace-Segment-Support/umb-protocol where the Avro schema is now maintained and pre-generated into Java. The MaaS, Salus, and UIM projects are all using umb-protocol.